src/poggit/webhook/DockerWebhookModule.php

```diff
@@ -0,0 +1,68 @@
+<?php
+
+/*
+ * Poggit
+ *
+ * Copyright (C) 2016-2020 Poggit
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace poggit\webhook;
+
+use poggit\Meta;
+use poggit\module\Module;
+use poggit\utils\lang\Lang;
+
+class DockerWebhookModule extends Module {
+    public function output() {
+        if($_SERVER['REQUEST_METHOD'] !== 'POST' or !isset($_GET['key'])){
+            $this->errorNotFound();
+        }
+
+        if($_GET['key'] !== Meta::getSecret("docker.webhookSecret")){
+            $this->errorAccessDenied();
+        }
+
+        Meta::getLog()->v("Docker update data received.");
```

Comment:
This duplicates the generic request log in verbose. Consider removing.

Reply:
Was to differentiate if it was 'real' see checks before